.circleci/marvin-analyze.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -e
```

Comment:
why are you separating to `.sh` and `.js` file? 
I think it might be much clearer if everything was in javascript and `circle.yml` would just run node scripts.

Reply:
The distinction here is having environment plumbing happen in sh files (which are most concise and readable for that purpose) and having logic in the js files (handling json, calling the external `judge-dredd` package for test analysis, etc.)

You're right that there are 2 files per step, but I feel that merging the 2 types would make it all far less readable and testable.